packages/ra-ui-materialui/src/form/TabbedForm.js

```diff
@@ -72,121 +73,122 @@ export const getTabFullPath = (tab, index, baseUrl) =>
         tab.props.path ? `/${tab.props.path}` : index > 0 ? `/${index}` : ''
     }`;
 
-export class TabbedForm extends Component {
-    handleSubmitWithRedirect = (redirect = this.props.redirect) =>
-        this.props.handleSubmit(values => this.props.save(values, redirect));
+export const TabbedForm = ({
```

Comment:
Same remark concerning the 2 components as in Simpleform

Reply:
I don't get this comment